lms/djangoapps/discussion_api/serializers.py

```diff
@@ -290,6 +290,15 @@ class CommentSerializer(_ContentSerializer):
 
     non_updatable_fields = NON_UPDATABLE_COMMENT_FIELDS
 
+    def __init__(self, *args, **kwargs):
+        remove_fields = kwargs.pop('remove_fields', None)
```

Comment:
Is there a reason to prefer `kwargs.pop` over including `remove_fields=None` in the `__init__` signature?